microsoft-edge/devtools-guide-chromium/beginners/css.md

```diff
@@ -590,11 +590,11 @@ Complete the following actions to bootstrap and provide classes for aligning ele
     
 ## Next steps  
 
-Congratulations, you are done.  
+Congratulations, you're done.  
```

Comment:
```suggestion
Congratulations, you're done!
```
Suggested for natural tone in this context.  https://docs.microsoft.com/en-us/style-guide/punctuation/exclamation-points